src/hub.js

```diff
@@ -819,83 +820,99 @@ document.addEventListener("DOMContentLoaded", async () => {
     mediaSearchStore
   );
 
+  let isInitialJoin = true;
+  let isInitialSync = true;
   hubPhxChannel
     .join()
     .receive("ok", async data => {
       hubChannel.setPhoenixChannel(hubPhxChannel);
 
+      socket.params().session_id = data.session_id;
+      socket.params().session_token = data.session_token;
+
       const hubPhxPresence = hubChannel.presence;
-      let isInitialSync = true;
       const vrHudPresenceCount = document.querySelector("#hud-presence-count");
 
-      hubPhxPresence.onSync(() => {
-        remountUI({ presences: hubPhxPresence.state });
-        const occupantCount = Object.entries(hubPhxPresence.state).length;
-        vrHudPresenceCount.setAttribute("text", "value", occupantCount.toString());
+      if (isInitialJoin) {
```

Comment:
Mostly just wrapped this block in an if statement. Take a look at the diff with whitespace ignored.